src/main/java/com/intuit/graphql/orchestrator/stitching/XtextStitcher.java

```diff
@@ -119,6 +124,14 @@ public RuntimeGraph stitch(List<ServiceProvider> serviceProviders) {
 
     });
 
+    stitchedTransformedGraph.getEntityExtensionDefinitionMap().keySet().forEach(typeName-> {
```

Comment:
for each on map instead of keyset

Reply:
used for each map [here](EntityBatchLoader entityBatchLoader = EntityBatchLoader.builder()
              .entityExtensionMetadata(entityExtensionMetadata)
              .build();
          batchLoaders.put(entityExtensionMetadata.getDataLoaderKey(), entityBatchLoader);)

Reply:
used for each map [here](https://github.com/intuit/graphql-orchestrator-java/pull/47/files#diff-f4115a8870c8e4e303736c7cdbbb0eef2b6ca0a2aef6a39f3e5ab76431949f33R127)